src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -79,3 +80,325 @@ def cli(self, output=None):
 
         return ret_dict
 
+
+# ==========================
+# Schema for 'show snmp'
+# ==========================
+class ShowSnmpSchema(MetaParser):
+
+    ''' Schema for "show snmp mib" '''
+
+    schema = {
+        "chassis": str,
+        "contact": str,
+        Optional("location"): Optional(str),
+        "snmp_input": {
+            "packet_count": int,
+            "bad_snmp_version_errors": int,
+            "unknown_community_name": int,
+            "illegal_operation_for_community_name_supplied": int,
+            "encoding_errors": int,
+            "number_of_requested_variables": int,
+            "number_of_altered_variables": int,
+            "get_request_pdus": int,
+            "get_next_pdus": int,
+            "set_request_pdus": int,
+            "input_queue_drops": int
+        },
+        "snmp_output": {
+            "packet_count": int,
+            "too_big_errors": int,
+            "no_such_name_errors": int,
+            "bad_value_errors": int,
+            "general_errors": int,
+            "response_pdus": int,
+            "trap_pdus": int
+        },
+        Optional("snmp_input_queue"): Optional(int),
+        Optional("snmp_global_trap"): Optional(str),
+        "snmp_logging" : {
+            "status": str,
+            Optional("endpoints") : {
+                Optional("ip_address"): {
+                    Optional(Any()) : {
+                        Optional("port") : Optional(int),
+                        Optional("buffer"): Optional(str),
+                        Optional("sent"): Optional(int),
+                        Optional("dropped"): Optional(int)
+                    }
+                }
+            }
+        }
+    }
+
+# ==========================
+# Parser for 'show snmp'
+# ==========================
+class ShowSnmp(ShowSnmpSchema):
+
+    ''' Parser for "show snmp" '''
+
```

Comment:
Don't have to on this one, as I don't see a big use case for comparing this, but consider using the exclude key as seen here: `https://github.com/CiscoTestAutomation/genieparser/blob/213a1f32f152b9e029864341cec64f8e7716ace3/src/genie/libs/parser/viptela/show_bfd_sessions.py#L45`

Reply:
Added.